docs/api-docs/store-management/shipping/use-real-time-carrier.md

```diff
@@ -0,0 +1,375 @@
+
+![Connected vs Installed Carrier](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Installed%20vs%20Connected%20Carrier.png 'Connected versus Installed Carrier. USPS is connected, while FedEx is only installed.')  
+
+### Create a connection
+
+To set up a carrier using the API, connect it using the [Create a Carrier Connection](/api-reference/store-management/shipping-api/shipping-carrier/postshippingcarrierconnection) endpoint. Make a request containing the ID of the carrier and any configuration fields that the carrier requires. All configuration fields are unique to each carrier. 
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example POST request with X-Auth-Token header" lineNumbers
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{	
+  "carrier_id" : "endicia",
+  "connection": {
+      "account_id" : "example_id",
+      "pass_phrase" : "example_passphrase"
+  }
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example POST response" lineNumbers
+No content
+```
+
+<!-- type: tab-end -->
+
+If a carrier doesn’t require any connection settings, send an empty object for the `connection` property.
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example POST request with X-Auth-Token header" lineNumbers
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "carrier_id": "example_carrier",
+  "connection": {}
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example POST response" lineNumbers
+No content
+```
+
+<!-- type: tab-end -->
+
+### Update a connection
+
+When you update a connection, use the same fields for a carrier as [Create a connection](#create-a-connection). 
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example PUT request with X-Auth-Token header" lineNumbers
+PUT https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{	
+  "carrier_id" : "endicia",
+  "connection": {
+      "account_id" : "example_id_2",
+      "pass_phrase" : "example_passphrase"
+  }
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example PUT response" lineNumbers
+No content
+```
+
+<!-- type: tab-end -->
+
+### Delete a connection
+
+To delete a connection, specify `carrier_id` in the request body.
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example DELETE request with X-Auth-Token header" lineNumbers
+DELETE https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "carrier_id": "example_carrier"
```

Comment:
```suggestion
  "carrier_id": "<<whatever-kind-of-string-a-carrier-id-typically-is>>"
```
consider this.

Reply:
The carrier_id is usually the name of the carrier (fedex, endicia, etc)